private-endpoint.tf

```diff
@@ -10,17 +10,10 @@ resource "azurerm_private_endpoint" "postgres" {
     subresource_names              = ["postgresqlServer"]
     is_manual_connection           = false
   }
-  count = var.subnet_id == "" ? 0 : 1
-}
-
-resource "azurerm_private_dns_a_record" "postgres" {
-  provider = azurerm.private_dns
-
-  name                = azurerm_postgresql_server.postgres-paas.name
-  zone_name           = "privatelink.postgres.database.azure.com"
-  resource_group_name = "core-infra-intsvc-rg"
-  ttl                 = 300
-  records             = [azurerm_private_endpoint.postgres[0].private_service_connection[0].private_ip_address]
 
+ private_dns_zone_group {
```

Comment:
can you also remove https://github.com/hmcts/cnp-module-postgres/blob/postgresql_tf/provider.tf#L1-L3

Reply:
^ changes updated